aloha/robot_utils.py

```diff
@@ -1,175 +1,102 @@
 from collections import deque
 import time
 from typing import Sequence
+import os
+import yaml
 
-from aloha.constants import (
-    COLOR_IMAGE_TOPIC_NAME,
-    DT,
-    IS_MOBILE,
-)
 from cv_bridge import CvBridge
 from interbotix_xs_modules.xs_robot.arm import InterbotixManipulatorXS
 from interbotix_xs_modules.xs_robot.gravity_compensation import (
     InterbotixGravityCompensationInterface,
 )
 from interbotix_xs_msgs.msg import JointGroupCommand, JointSingleCommand
 import numpy as np
+from pathlib import Path
 from rclpy.node import Node
 from sensor_msgs.msg import Image, JointState
 
 
 class ImageRecorder:
     def __init__(
         self,
-        is_mobile: bool = IS_MOBILE,
+        config: dict,
         is_debug: bool = False,
         node: Node = None,
     ):
         self.is_debug = is_debug
         self.bridge = CvBridge()
 
-        if is_mobile:
-            self.camera_names = ['cam_high', 'cam_left_wrist', 'cam_right_wrist']
-        else:
-            self.camera_names = ['cam_high', 'cam_low', 'cam_left_wrist', 'cam_right_wrist']
+        camera_config = config.get('cameras', {})
+        # Get camera names from config dictionary
+        self.camera_names = [camera['name']
+                             for camera in camera_config.get('camera_instances', [])]
+
+        color_image_topic_name = config.get('cameras', {}).get(
+            'common_parameters', {}).get('color_image_topic_name', None)
```

Comment:
Should we assert or throw that these configs have been set to not `None`?

Reply:
Will do that.